draft-ietf-mptcp-rfc6824bis-11.xml

```diff
@@ -420,21 +420,23 @@ to allow subflows to be created when either end is behind a NAT, MPTCP uses the
         so on this particular connection.</t>
 
         <t>The MP_CAPABLE exchange in this specification (v1) is different to
-        that specified in v0 <xref target="RFC6824"/>. 
-        If a host supports multiple versions of MPTCP, the sender of the
-        MP_CAPABLE option SHOULD signal the highest version number it supports.
-        The passive opener, on receipt of this, will signal the version number
-        it wishes to use, which MUST be equal to or lower than the version 
-        number indicated in the initial MP_CAPABLE. 
-        Given the SYN exchange is different between v1 and v0 the exchange cannot
-        be immediately downgraded, and therefore if the far end has requested a 
-        lower version then the initiator SHOULD respond with an ACK without any
-        MP_CAPABLE option, to fall back to regular TCP. If the initiator supports
-        the requsted version, on future connections to the target host, the initiator 
-        MAY cache the version preference. Alternatively, the initiator MAY close 
-        the connection with a TCP RST and immediately re-establish with the requested
-        version of MPTCP.</t>
-        
+        that specified in v0 <xref target="RFC6824"/>.  If a host supports multiple versions
+        of MPTCP, the sender of the MP_CAPABLE option SHOULD signal the
+        highest version number it supports.  The passive opener, on receipt
+        of this, will signal the version number it wishes to use, which MUST
+        be equal to or lower than the version number indicated in the initial
+        MP_CAPABLE.
+        There is a caveat though with respect to this version negotiation with
+        old listeners that only support v0. A listener that supports v0 expects that
+        the MP_CAPABLE option in the SYN-segment includes the initiator's key. If
+	the initiator however already upgraded to v1, it won't include the key in the
+        SYN-segment. Thus, the listener will ignore the MP_CAPABLE of this
+        SYN-segment and reply with a SYN/ACK that does not include an MP_CAPABLE.
+        The initiator MAY retry a new connection with MPTCP v0 or regular TCP, based
```

Comment:
This could be read as saying the connection must be re-established. Better wording may be:

...that does not include an MP_CAPABLE, thus leading to a fallback to regular TCP. An initiator MAY cache this information about a peer and for future connections, MAY choose to attempt using MPTCP v0, if supported, before recording the host as not supporting MPTCP.</t>

Reply:
Sounds good! I will rephrase this.